modules/importing-repositories/procedure-bulk-import-scaffolder-template.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="procedure-bulk-import-scaffolder-template"]
+= Understanding Bulk Import Scaffolder template input parameters
+
+You can use the Bulk Import plugin to run a Scaffolder template task with specified parameters. As an administrator, you must use these parameters within the template.
+
+The Bulk Import plugin analyzes Git repository information and provides the following parameters for the Scaffolder template task:
+
+`repoUrl`:: Normalized repository URL in the following format:
++
+[source,yaml]
+----
+  ${gitProviderHost}?owner=${owner}&repo=${repository-name}
+----
++
+For example, when you run the process, the plugin transforms `https://github.com/redhat-developer/rhdh-plugins` into `github.com?owner=redhat-developer&repo=rhdh-plugins`.
```

Comment:
I wonder if it's necessary to include the example of `https://github.com/redhat-developer/rhdh-plugins` throughout this module since it's quite self-explanatory